easybuild/easyconfigs/v/VASP/VASP-5.4.4-18Apr17-p01-intel-2018b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = '5.4.4-18Apr17-p01'
+
+homepage = 'http://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale
+materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics,
+from first principles."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+# Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
+sources = ['%(namelower)s.%(version)s.tar.gz']
+
+# patch from Vaspwiki, http://cms.mpi.univie.ac.at/patches/patch.5.4.4.16052018.gz
+patches = ['vasp.5.4.4.16052018.patch']
+
+checksums = [
+    '5bd2449462386f01e575f9adf629c08cb03a13142806ffb6a71309ca4431cfb3',  # vasp.5.4.4.tar.gz
+    '3392a0ffccfa17a77775be86b1bed74f8f75a0a642f77ee0fae59950d0a3d3f4',  # vasp.5.4.4.16052018.patch
+]
+
+prebuildopts = 'cp arch/makefile.include.linux_intel ./makefile.include && '
+
```

Comment:
Are you sure you don't need this?
```
# path to libfftw3xf_intel.a is hardcoded in makefile.include
prebuildopts += 'sed -i "s|\$(MKLROOT)/interfaces/fftw3xf|\$(FFTW_LIB_DIR)|" makefile.include && '

# remove mkl flag to prevent mixing dynamic libs with the static libs in LIBBLACS/SCALAPACK
prebuildopts += 'sed -i "s|-mkl[^ ]*||" makefile.include && '
```

Reply:
@akesandgren I think the FFTW3 wrappers have been included in the main MKL library (although not exactly sure as to since which version of MKL)

since the "-mkl=sequential" in the Makefile is in the "FCL" definition, redefining it clears it, but yes, if you don't redefine FCL you'd still need to sed it out